certification/internal/utils/file/helper.go

```diff
@@ -48,3 +48,10 @@ func Unzip(bzipfile string, destination string) error {
 	out.Close()
 	return nil
 }
+
+func HomeDir() string {
+	if h := os.Getenv("HOME"); h != "" {
+		return h
+	}
+	return os.Getenv("USERPROFILE") // windows
```

Comment:
Is this really necessary? Is this something we anticipate having to support?

Reply:
I think I would prefer requiring an explicitly defined kubeconfig location. @samira-barouti what do you think?

Reply:
I removed this new function and leveraged viper to look for kubeconfig under $HOME/.kubee/config as the default location.